index.html

```diff
@@ -0,0 +1,210 @@
+
+New for 2023, experience an unparalleled atmosphere and an unbeatable view on all the action – THE SUITES at SINGAPORE SMASH – your bespoke, high-end table tennis experience.<br>
+
+Enjoy first-class entertainment and adrenaline-filled performances with staff or clients, all from an immersive fully serviced corporate box located above the player walkout runway.
+        </p>
+        <p class="date">2023.10.13(WED) ~ 15(FRI)</p>
+        <p class="place">@Singapore, Singapore.</p>
+      </div>
+    </div>
+
+    <section class="main-program" id="program">
+      <div class="main-title">
+        <h3 class="main-title-h3">THE SUITES</h3>
+      </div>
+      <div class="main-bar"></div>
+      <div class="main-activities">
+        <table class="table table-borderless t-main">
+          <tbody>
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_01.png" alt="Speaker Icon"></td>
+              <td class="t-title">Full - Service Lounge</td>
+              <td class="t-description">Branded, private, full-service lounge with food beverages</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <table class="table table-borderless t-main">
+          <tbody class="main-column">
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_02.png" alt="Exhibit Icon"></td>
+              <td class="t-title">Unique Experiences</td>
+              <td class="t-description">Visit legends and current stars in the arena.</td>
+            </tr>
+          </tbody>
+        </table>
+  
+        <table class="table table-borderless t-main">
+          <tbody>
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_03.png" alt="Forum Icon"></td>
+              <td class="t-title">Special Perks</td>
+              <td class="t-description">Backstage tour, training room, memorabilia and more.</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <table class="table table-borderless t-main">
+          <tbody>
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_04.png" alt="Workshop Icon"></td>
+              <td class="t-title">Great Views</td>
+              <td class="t-description">Private siting.</td>
+            </tr>
+          </tbody>
+        </table>
+  
+       
+
+      </div>
+
+      <div class="joinButton" id="join">
+        <button class="join">
+          Join Singapore Smash 2023
+        </button>
+      </div>
+
+      <div class="program">
+        <a class="see-program" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+      
+
+    <section class="speakers" id="speakers">
+      <div class="featured">
+        <h4>Top Players</h4>
+      </div>
+      <div class="main-bar"></div>
+        
+      <div class="grid-container">
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, in your top players section your images are not showing and your margin and text style does not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your design
![image](https://user-images.githubusercontent.com/92303891/227748521-13c7965a-0093-4cc8-b6a6-6c4b2aa0d3dd.png)


### Cindy's design
![Screenshot from 2023-03-26 00-07-44](https://user-images.githubusercontent.com/92303891/227748375-89932e9b-bc55-4e4a-ba99-b50f8939a825.png)